data/ka'het/ka'het missions.txt

```diff
@@ -50,15 +53,22 @@ mission "First Contact: Ka'het: Remnant 1A"
 				`	(Yes.)`
 				`	(Not now.)`
 					defer
+			branch Taely
+				has "remnant met taely"
```

Comment:
This mission doesn't offer if you have this condition, so this branch can never execute. I think a better course of action would be to allow Remnant 1B to offer anywhere.
Labels should be lowercase.

Reply:
Changed not to care about having "remnant met taely" or not, as that is not unnecessary since it has been changed not to offer on Viminal (there is the case of the player landing on viminal without having met taely and not having the mission, but this will pop up as soon as they land on another remnant planet so it doesn't seem problematic to me).
I prefer to still have this as the non-viminal mission and keep 1B only there, as as far as the player is concerned that is the place where Taely is

Reply:
Now a player who hasn’t met Taely but has found the Graveyard will get no information when landing in Viminal.

Reply:
Added a third mission for Viminal